Misc/NEWS.d/next/Library/2017-11-10-00-05-08.bpo-31993.-OMNg8.rst

```diff
@@ -0,0 +1,3 @@
+The C-based and Python-based picklers do no longer allocate temporary memory
```

Comment:
There are no other picklers except the C-based and Python-based picklers. :smile_cat: